test/DotNetEnv.Tests/ParserTests.cs

```diff
@@ -340,6 +366,15 @@ public void ParseAssignment ()
                 Assert.Equal(value, kvp.Value);
             };
 
+            testParse("EV_DNE", "", "EV_DNE=#no value just comment");
```

Comment:
These new tests all seem excellent, and (other than the 2 that already pass) seem to directly test the desired new behavior.